extensions/elasticsearch-java-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,14 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Elasticsearch Java Client"
+metadata:
+  keywords:
+  - "elasticsearch"
+  - "full text"
+  - "search"
+  guide: "https://quarkus.io/guides/elasticsearch"
+  categories:
+  - "data"
+  status: "preview"
+  config:
+  - "quarkus.elasticsearch."
```

Comment:
This is the same prefix as for the low-level client and the existing high-level client... is this wise?

Reply:
Yes, it uses the exact same configuration as the Java Client is build using the Rest client as a transport.